app/models/concerns/helper.rb

```diff
@@ -0,0 +1,10 @@
+require 'active_support/concern'
```

Comment:
I would rename `Helper` into something more meaningful. A concern is supposed to be reusable behaviour so having a good name is really important. 

As a side note, I am not sure of using a concern here. My feeling is that we are just moving code around. It is not adding much value. Indeed, you can almost inline the calculation within a variable in the method that is being used. Much simpler with less ceremony.

Second side note: I would have also expected a test for the concern `helper_spec`. But still, the name feels not right to me.